ext/build_defaults.yaml

```diff
@@ -0,0 +1,28 @@
+---
+project: 'puppet-runtime'
+packaging_url: 'git://github.com/puppetlabs/packaging.git --branch=master'
+packaging_repo: 'packaging'
+# foss_platforms will be shipped to the nightly repos
+# foss_platforms:
+pe_platforms:
+  - el-6-x86_64
+  - el-7-x86_64
+  - osx-10.11-x86_64
+  - osx-10.12-x86_64
+  - sles-11-x86_64
+  - sles-12-x86_64
+  - ubuntu-14.04-amd64
+  - ubuntu-16.04-amd64
+  - windows-2012-x64
+gpg_name: 'info@puppetlabs.com'
+gpg_key: '7F438280EF8D349F'
+deb_targets: 'trusty-amd64 xenial-amd64'
+rpm_targets: 'el-6-x86_64 el-7-x86_64 sles-11-x86_64 sles-12-x86_64'
+sign_tar: FALSE
+osx_signing_cert: "Developer ID Installer: PUPPET LABS, INC. (VKGLGN2B6Y)"
+osx_signing_keychain: "/Users/jenkins/Library/Keychains/signing.keychain"
+osx_signing_server: 'osx-signer.delivery.puppetlabs.net'
+vanagon_project: TRUE
+apt_repo_name: 'PC1'
+yum_repo_name: 'PC1'
```

Comment:
Do we want these still?

Reply:
Copied from PDK. Commit from yesterday was reverted as it broke things, can copy it over when it's fixed. These don't really matter at all though since we aren't generating packages and aren't shipping artifacts publicly

Reply:
yeah, I was just hoping to avoid having lots of unnecessary things, since I think it's easy to cause confusion.